.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,16 +5,25 @@ about: Create a report to help reproduce and fix the issue
 <!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable -->
 ## Round ID:
 
-<!--- **INCLUDE THE ROUND ID**
-If you discovered this issue from playing tgstation hosted servers:
-[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)-->
+<!--- **INCLUDE THE ROUND ID** [Round ID]: # (It can be found in the Status panel -->
 
-## Testmerges:
+## Date and Time of Issue:
+
+<!-- When did you encounter this issue for the first time. Make sure to mention your timezone when listing the Date and Time -->
```

Comment:
This should either be using the standard timezone of the server location (UTC-5, EST) for conformity with logs.